src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -85,9 +85,9 @@ export const content: GettingStartedContent = [
 				{
 					id: 'remoteTerminal',
 					title: localize('gettingStarted.remoteTerminal.title', "Run tasks in the integrated terminal"),
-					description: localize('gettingStarted.remoteTerminal.description', "Access your full development environment in the cloud and perform quick command-line tasks."),
+					description: localize('gettingStarted.remoteTerminal.description', "Perform quick command-line tasks using the built-in terminal."),
 					button: {
-						title: localize('gettingStarted.remoteTerminal.button', "Focus Terminal"),
+						title: localize('gettingStarted.remoteTerminal.button', "Open Terminal"),
```

Comment:
Terminal is open by default, which is why I ended up with (slightly odd) _"Focus"_. Open works as well though, just might be slightly confusing.

Reply:
Ah, I see now. I think "Focus" makes sense there then! I saw you merged this PR, so do you want to just update this one?